src/codeMirror6Scroller.ts

```diff
@@ -0,0 +1,31 @@
+import type { ContentScriptContext, MarkdownEditorContentScriptModule } from 'api/types';
+import { EditorView } from '@codemirror/view';
+
+// modified from: https://github.com/personalizedrefrigerator/bug-report/tree/example/plugin-scroll-to-line
+export default (context: ContentScriptContext): MarkdownEditorContentScriptModule => {
+	return {
+		plugin: (editorControl: any) => {
+			if (!editorControl.cm6) { return; }
+
+      // Running in CM6
+      editorControl.registerCommand('scrollToLine', (lineNumber: number) => {
+        const editor: EditorView = editorControl.editor;
+        console.log('scrollToLine', lineNumber);
+
+        // Bounds checking
+        if (lineNumber < 0) {
+          lineNumber = 0;
+        }
+        if (lineNumber > editor.state.doc.lines) {
+          lineNumber = editor.state.doc.lines;
+        }
+
+        // Scroll to line, place the line at the *top* of the editor
+        const lineInfo = editor.state.doc.line(lineNumber);
+        editor.dispatch(editor.state.update({
+            effects: EditorView.scrollIntoView(lineInfo.from, {y: 'start'})
+        }));
+      });
+		},
```

Comment:
@alondmnt This currently uses mixed tabs and space indentation.

(Thank you for working on this!)

Also note that if upgrading the plugin API proves to be too big of a change for this pull request, it should be possible to require CodeMirror 6 modules (with no types) with `joplin.require`. For example,
```
const { EditorView } = joplin.require('@codemirror/view');
```